Readme.md

```diff
@@ -81,6 +82,7 @@ modules:
         - thedevelopers
       create: true
       delete: true
+      reset: true
```

Comment:
I added the boolean enable to complement create/delete, but really reset is a command line flag. The way I have it set up, the boolean flag needs to be true before -reset *uid* will work. This is gratuitous, but follows the style of the create/delete booleans. Thoughts?

I think it would be reasonable to make it only a command line flag.